releasenotes-builder/src/main/java/com/github/checkstyle/Main.java

```diff
@@ -66,6 +70,26 @@ public static void main(String... args) {
                 final Result notesBuilderResult = runGithubNotesBuilder(cliOptions);
                 errorCounter = notesBuilderResult.getErrorMessages().size();
                 if (errorCounter == 0) {
+                    final char[] charArray = cliOptions.getReleaseNumber().toCharArray();
+                    final long numberOfPeriods = IntStream
+                        .range(0, charArray.length)
+                        .mapToObj(i -> charArray[i])
+                        .filter(ch -> ch == '.')
+                        .count();
```

Comment:
be simple:
```
String someString = "elephant";
long count = someString.chars().filter(ch -> ch == 'e').count();
assertEquals(2, count);
```

Reply:
We can't use assertEquals here? releasenotes-builder doesn't recognize it.

Reply:
It is quick copy paste from web.
Use `someString.chars().filter(ch -> ch == 'e').count();` with modification to serve your need

Reply:
Changes Done